iron-selection.html

```diff
@@ -32,7 +32,7 @@
      * the selected item or undefined if there is no selection.
      */
     get: function() {
-      return this.multi ? this.selection : this.selection[0];
+      return this.multi ? this.selection.slice() : this.selection[0];
```

Comment:
Is this still necessary with the updates made to https://github.com/PolymerElements/iron-selector/pull/48/files#diff-5c8329bac0158c065b259270a43cfe7bR117 ?